src/test/test-integration.ts

```diff
@@ -34,128 +32,156 @@ describe("Integration test between client and service", function(){
     };
     client.onMessage(this._boardService.onClientConnection());
     this._clients.push(client);
-  }
+  };
 }
 
-  it("Test with one client, case one: change, client tic, change server tic.", function(){
-      // Arrange
-        // Prepare variables
-      var boardVM: BoardVM = new BoardVM();
-      var client = new App(boardVM);
-      var noteVM = boardVM.createNote();
-      var beginingText = "This is the begining part of the test";
-      var endingText = " and this is the ending part of the test";
-      var boardService = new BoardService();
-      var serverMock = new ServerMock(boardService);
-      serverMock.addClient(client);
-
-      // Act
-      noteVM.content(beginingText);
-      client.onInterval();
-      noteVM.content(noteVM.content() + endingText);
-      boardService.onTic();
-
-      //Assert
-      expect(noteVM.content()).toEqual(beginingText + endingText);
-    });
-	
-  it("Test with one client, case two: change, client tic, change, client tic, server tic.", function(){
-      // Arrange
-        // Prepare variables
-      var boardVM: BoardVM = new BoardVM();
-      var client = new App(boardVM);
-      var noteVM = boardVM.createNote();
-      var beginingText = "This is the begining part of the test";
-      var endingText = " and this is the ending part of the test";
-      var boardService = new BoardService();
-      var serverMock = new ServerMock(boardService);
-      serverMock.addClient(client);
-
-      // Act
-      noteVM.content(beginingText);
-      client.onInterval();
-      noteVM.content(noteVM.content() + endingText);
-      client.onInterval();
-      boardService.onTic();
-
-      //Assert
-      expect(noteVM.content()).toEqual(beginingText + endingText);
-    });
-
-  it("Test with one client, case three: change, client tic, server tic, change, client tic, server tic.", function(){
-      // Arrange
-        // Prepare variables
-      var boardVM: BoardVM = new BoardVM();
-      var client = new App(boardVM);
-      var noteVM = boardVM.createNote();
-      var beginingText = "This is the begining part of the test";
-      var endingText = " and this is the ending part of the test";
-      var boardService = new BoardService();
-      var serverMock = new ServerMock(boardService);
-      serverMock.addClient(client);
-
-      // Act
-      noteVM.content(beginingText);
-      client.onInterval();
-      boardService.onTic();
-      noteVM.content(noteVM.content() + endingText);
-      client.onInterval();
-    boardService.onTic();
+// describe("Integration tests with one client", function(){
```

Comment:
it shouldn't be commented. I was some issues with versioning. I'll push the last version.